packages/base/src/util/dragAndDrop/DragRowUtil.ts

```diff
@@ -0,0 +1,98 @@
+import type UI5Element from "../../UI5Element.js";
+import type MovePlacement from "../../types/MovePlacement.js";
+import DragRegistry from "./DragRegistry.js";
+import { findClosestPosition } from "./findClosestPosition.js";
+import Orientation from "../../types/Orientation.js";
+
+type DragOverResult = {
+	targetReference: HTMLElement | null;
+	placement: any;
+}
+
+/**
+ * Drags an element over a list of items and returns the target reference and the placement of the dragged element.
+ *
+ * @param e {DragEvent} The drag event.
+ * @param component {UI5Element} The component, which contains the items.
+ * @param items {Array<HTMLElement>} The items, which the dragged element is dragged over.
+ * @returns {DragOverResult} The target reference and the placement of the dragged element.
+ */
+function dragOver(e: DragEvent, component: UI5Element, items: Array<HTMLElement>): DragOverResult {
+	const dragOverResult: DragOverResult = {
+		targetReference: null,
+		placement: null,
+	};
+
+	const draggedElement = DragRegistry.getDraggedElement();
+
+	if (!(e.target instanceof HTMLElement) || !draggedElement) {
+		return dragOverResult;
+	}
+
+	const closestPosition = findClosestPosition(
+		items,
+		e.clientY,
+		Orientation.Vertical,
+	);
```

Comment:
Although this code is the same for List and Table, it is not for the Tree and TabContainer. Maybe put this back to the components and only reuse the code below?

Reply:
Moved `closestPosition` parts to components. Kept `draggedElement` in the util